apache/superset
# chore: Remove unused less file from profile

### SUMMARY
It removes the unused main.less file.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
No visual changes

### TESTING INSTRUCTIONS
1. Visit the profile
2. Make sure there are no visual changes

### ADDITIONAL INFORMATION
